src/components/Horizontal.jsx

```diff
@@ -0,0 +1,53 @@
+import { FaStar, FaRegStar, FaTimes, FaStarHalfAlt, FaPlus, FaMinus } from "react-icons/fa"
+import { useProducts } from "../Context/ProductContext"
+const Horizontal = ({ brand, category, discount, gender, inStock, name, price, rating, _id, img, alt, quantity}) => {
```

Comment:
suggestion: instead of passing many props. you can pass them via a single object like **product={product}**